libkbfs/cr_chains_test.go

```diff
@@ -186,7 +186,7 @@ func TestCRChainsSingleOp(t *testing.T) {
 
 	co, err := newCreateOp("new", dir2Unref, File)
 	require.NoError(t, err)
-	currPtr = testCRFillOpPtrs(currPtr, expected, revPtrs,
+	_ = testCRFillOpPtrs(currPtr, expected, revPtrs,
```

Comment:
I don't think it enforces that, i.e. we can drop the assignment entirely. I just went with it as the safe option.